### DISCUSSION TITLE
Help us name the 0.15.0 release!

### DISCUSSION THREAD

**Discussion by molliemarie** [dagster-io/dagster#8234] [Community]
Hello Dagster Community!

We are excited to be releasing version 0.15.0 of Dagster soon, and we want your help naming the release! :D 

If you’ve paid attention to our release names to date, you may have noticed a trend in our naming scheme, which is that the release names tend to line up with the songs or albums of pop divas. Check out some of our past release names:

* [Never Felt Like This Before](https://www.youtube.com/watch?v=qXNPvF6v_VU) (Shaznay Lewis)
* [Get the Party Started](https://www.youtube.com/watch?v=mW1dbiD_zDk) (Pink)
* [Into the Groove](https://www.youtube.com/watch?v=52iW3lcpK5M) (Madonna)
* [Lucky Star](https://www.youtube.com/watch?v=ThHz9wlBeLU) (Madonna)
* [The Edge of Glory](https://www.youtube.com/watch?v=QeWBS0JBNzQ) (Lady Gaga)
* [Laundry Service](https://www.youtube.com/watch?v=7BPE5sR6oUU&list=PL035E8452800ACEA5) (Shakira)
* [In the Zone](https://www.youtube.com/watch?v=we8h_2RFAyA) (Britney Spears)
* [Waiting to Exhale](https://www.youtube.com/watch?v=_FT7ciAPuJk&list=PL_sN_rrP4od0gZ4C3LGkgcaRpVa_koQX4) (Whitney Housten, et al.)
* [Impossible Princess](https://www.youtube.com/playlist?list=PL6sQOXR9ddaobFB1B1dwFLRbHO9O8qxvk) (Kylie Minogue)

In the past, we’ve sourced a list of names internally and then voted internally to come up with the name. But this time we want you to help us! In the comments below you'll see the names that have been sourced by the Dagster team. Do us a favor and vote on your favorite by adding a ❤️ . 

In case you want a bit of extra context: the headline of Dagster 0.15.0 is that software-defined assets are now marked fully stable and are ready for primetime. We will have a blog post detailing more about that, as well as all the other updates, coming soon. 

Thanks for being a part of our release naming! :D

Cheers,

----------
_**TL;DR: Vote on your favorite release name by reacting to the comments below:**_

**Comment by molliemarie** [comment]
New Rules

**Comment by molliemarie** [comment]
Fifteen

**Comment by molliemarie** [comment]
Dressed to Kill

**Comment by molliemarie** [comment]
What You Waiting For

**Comment by molliemarie** [comment]
Cool for the Summer

**Comment by molliemarie** [comment]
Roar

**Comment by molliemarie** [comment]
Wildest Dreams

**Comment by molliemarie** [comment]
Supa Dupa Fly

**Comment by molliemarie** [comment]
Tapestry

**Comment by molliemarie** [comment]
Control

**Comment by molliemarie** [comment]
This Time Baby

**Comment by molliemarie** [comment]
Smooth Operator

**Comment by molliemarie** [comment]
Hella Good
